usr/share/lib/img_proof/tests/SLES/EC2/test_sles_ec2_dracut_conf.py

```diff
@@ -0,0 +1,6 @@
+def test_sles_dracut_conf(host):
+    needed_drivers = ('ena', 'nvme', 'nvme-core', 'virtio', 'virtio_scsi',
+                      'xen-blkfront', 'xen-netfront')
+    dracut_conf = host.file('/etc/dracut.conf.d/07-aws-type-switch.conf')
+    for driver in needed_drivers:
+        assert dracut_conf.contains(drivers)
```

Comment:
/s/drivers/driver

Reply:
Typo fixed. But again I am happy to pull this if you think the loop is not OK, the close was just intended to not leave this dangling for investigation into pytest behavior with respect to network connection opening. Alternative to parameterize and avoid the investigation into pytest

```import pytest

dracut_conf = ''

@pytest....
def test_sles_dracut_conf(host,driver):
    global dracut_conf
    if not dracut_conf:
         dracut_conf = host.file('/etc/dracut.conf.d/07-aws-type-switch.conf')
    assert dracut_conf.contains(driver)
```

should work, but that has the global which is not particularly pretty either